src/test/java/org/llorllale/cactoos/matchers/FuncAppliesTest.java

```diff
@@ -36,10 +36,10 @@
  * @checkstyle JavadocMethodCheck (100 lines)
```

Comment:
@andreoss is this still needed? I think since the methods are not public anymore, the check does not fail

Reply:
@victornoel Removed those.

Reply:
@victornoel Btw, https://github.com/teamed/qulice/issues/1075

Reply:
@andreoss very nice :)